VisualPinball.Unity/VisualPinball.Unity.Editor/Inspectors/GateInspector.cs

```diff
@@ -31,12 +32,42 @@ public class GateInspector : ItemInspector
 		private static string[] _gateTypeStrings = { "Wire: 'W'", "Wire: Rectangle", "Plate", "Long Plate" };
 		private static int[] _gateTypeValues = { GateType.GateWireW, GateType.GateWireRectangle, GateType.GatePlate, GateType.GateLongPlate };
 
+		private static readonly string TwoWayLabel = "Two Way";
+
 		protected override void OnEnable()
 		{
 			base.OnEnable();
 			_gate = target as GateAuthoring;
 		}
 
+		protected virtual void OnSceneGUI()
+		{
+			if (target is IEditableItemAuthoring editable) {
+				var position = editable.GetEditorPosition();
+				var transform = (target as MonoBehaviour).transform;
+				if (transform != null && transform.parent != null) {
+					position = transform.parent.TransformPoint(position);
+					var axis = transform.TransformDirection(-Vector3.up); //Local direction of the gate gameObject is -up
+					var worldScale = 0.5f * VpxConverter.GlobalScale;
+					var scale = _gate.Item.Data.Length * worldScale;
+					Handles.color = Color.white;
+					Handles.DrawWireDisc(position, axis, scale);
+					Color col = Color.grey;
+					col.a = 0.25f;
+					Handles.color = col;
+					Handles.DrawSolidDisc(position, axis, scale);
+
+					Handles.color = Color.white;
+					var ratioscale = 2.0f;
+					var arrowscale = (worldScale * (100.0f - ratioscale)) + Mathf.PingPong(Time.realtimeSinceStartup * worldScale * ratioscale * 2.0f, worldScale * ratioscale);

```

Comment:
Can we get rid of the unused animation code?

Reply:
done

Reply:
With that it still animates, but only when the scene view camera moves, no?

Reply:
Hey @Vroonsh I think you still need to remove this one, then I'll merge!

Reply:
> With that it still animates, but only when the scene view camera moves, no?

yup exactly

Reply:
animation code removed for gates